en-US/Design.xml

```diff
@@ -767,7 +784,7 @@ STEP 14: COMMIT <emphasis>...output omitted...</emphasis> localhost/nexus:latest
 					<title>Choosing a Realistic Name</title>
 					 <para>
 						Consider the following points when choosing a realistic name: <footnote> <para>
-							Examples taken from the <citetitle>IBM Style Guide</citetitle> and the <citetitle>Google Developer Documentation Style Guide</citetitle>.
+							Examples taken from <citetitle>IBM Style</citetitle> and the <citetitle>Google Developer Documentation Style Guide</citetitle>.
```

Comment:
This footnote seems misplaced. Ideally, this footnote would appear just after the sentence where the examples are provided. However, this footnote is attached to a lead-in sentence that does not include any examples. As a result, it is unclear which material comes from these sources. This is likely out of scope for this PR, but consider moving this footnote to the sentence with the relevant examples. 

Could we phrase this more generically: "Examples adapted from" rather than "taken from"? I'm guessing that we didn't quote exact language from these sources, just adapted the example names for our discussion. But that's just a guess.

Reply:
I've changed how and where the reference information is included here. It seems that IBM Style does not provide specific guidance here; the Google developer documentation style guide does, although no specific examples are taken from there.